src/vm_stack.zig

```diff
@@ -2099,6 +2102,8 @@ const InstructionFuncs = struct {
             else => unreachable,
         };
 
+        std.debug.print("Growing from {} to {}", .{ old_num_pages, num_pages });
```

Comment:
For some reason, adding this debug line allows the mem64 tests to pass on Linux/windows/x86_64. Also running it with `-Doptimize=ReleaseSafe` without this statement allows it to pass as well. I think there might be a compilation issue, but need to dig into it a bit more.